conf/defaults.ini

```diff
@@ -911,6 +911,11 @@ zipkin_propagation = false
 # Not disabling is the most common setting when using Zipkin elsewhere in your infrastructure.
 disable_shared_zipkin_spans = false
 
+[tracing.opentelemetry.jaeger]
+enabled =
+# jaeger destination (ex http://localhost:14268/api/traces)
+address =
```

Comment:
Would it be easier to rely on a non-empty address here instead of the enabled flag? Like we do in `tracing.jaeger`? I can't see why enabled would be true and address be empty, or the other way around - non-empty address and enabled=false.

Reply:
I thought we could do this for in the future coming solutions other than jaeger, for me this makes more sense, but I am also open to changing it to the non-empty address

Reply:
ok. changed it